src/test_suite.js

```diff
@@ -12,6 +12,61 @@ TestSuite.prototype.name = function (name) {
   return this;
 };
 
+TestSuite.prototype.timestamp = function (timestamp) {
+  if(_.isDate(timestamp)) timestamp = timestamp.toISOString();
+  this._attributes.timestamp = timestamp;
+  return this;
+};
+
+TestSuite.prototype.time = function (time) {
+  this._attributes.time = time;
+  return this;
+};
+
+TestSuite.prototype.tests = function (tests) {
+  this._attributes.tests = tests;
+  return this;
+};
+
```

Comment:
Does the `tests` above and `testsAdd` below refer to the number of tests in the suite? Should this number always be the same as the number of `<error>` elements among the children of this suite?